src/itest/java/org/kiwiproject/consul/AclClientITest.java

```diff
@@ -24,49 +24,53 @@
 import java.util.List;
 import java.util.Objects;
 
+/**
+ * @implNote This does not extend {@link BaseIntegrationTest} because it needs to launch Consul with ACLs enabled
+ * and to create a client that uses the ACL token when connecting.
+ */
 class AclClientITest {
 
-    public static final GenericContainer<?> consulContainerAcl;
+    private static GenericContainer<?> consulContainerAcl;
+    private static Consul client;
+    private static HostAndPort aclClientHostAndPort;
+    private static AclClient aclClient;
 
-    static {
+    @BeforeAll
+    static void beforeAll() {
         // noinspection resource
         consulContainerAcl = new GenericContainer<>(CONSUL_DOCKER_IMAGE_NAME)
                 .withCommand("agent", "-dev", "-client", "0.0.0.0", "--enable-script-checks=true")
                 .withExposedPorts(8500)
                 .withEnv("CONSUL_LOCAL_CONFIG",
-                        "{\n" +
-                                "  \"acl\": {\n" +
-                                "    \"enabled\": true,\n" +
-                                "    \"default_policy\": \"deny\",\n" +
-                                "    \"tokens\": {\n" +
-                                "      \"master\": \"aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee\"\n" +
-                                "    }\n" +
-                                "  }\n" +
-                                "}"
+                        """
+                            {
```

Comment:
open/close the string symmetrically, with the three double quotes on the same line as the brace (or not)

Reply:
Will fix in separate commit